setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="vxvas2nets",
+    version="0.6.0a",
```

Comment:
I don't remember exactly, but I remember something about starting transport version numbers at the current vumi version number. Happy to change this to `0.0.1` or something like that though.